components/google_calendar/actions/create-acl/create-acl.js

```diff
@@ -0,0 +1,54 @@
+const googleCalendar = require("../../google_calendar.app");
+
+module.exports = {
+  key: "google_calendar_create_acl",
+  name: "Create an Access Control Rule for a calendar",
+  description: "Create an Access Control Rule for a calendar.",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    googleCalendar,
+    calendarId: {
+      propDefinition: [
+        googleCalendar,
+        "calendarId",
+      ],
+    },
+    role: {
+      propDefinition: [
+        googleCalendar,
+        "role",
+      ],
+    },
+    scopeType: {
+      propDefinition: [
+        googleCalendar,
+        "scopeType",
+      ],
+    },
+    scopeValue: {
+      propDefinition: [
+        googleCalendar,
+        "scopeValue",
+      ],
+    },
+  },
+  async run() {
+    const calendar = this.googleCalendar.calendar();
+    let scope = {
+      type: this.scopeType,
+    };
+
+    if (this.scopeType !== "default" && this.scopeValue.trim().length) {
+      scope["value"] = this.scopeValue;
+    }
+
+    return (await calendar.acl.insert({
+      calendarId: this.calendarId,
+      requestBody: {
+        role: this.role,
+        scope: scope,
+      },
+    })).data;
```

Comment:
I would recommend creating a new function named `insertAcl` in `google_calendar.app.js` like this 
```javascript
    async insertAcl(params) {
      const calendar = this.calendar();
      const { data } = await calendar.acl.insert(params);
      return data;
    },
```
following the same pattern as the other functions. Then you can call the function like this
```javascript
    return this.googleCalendar.insertAcl({
      calendarId: this.calendarId,
      requestBody: {
        role: this.role,
        scope: scope,
      },
    });
```